src/client/scripts/cardModal.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+/* global Cards:false SnackbarMethods:false */
+
+Template.cardModal.helpers({
+    cardModalInfo: function(_id) {
+        return Cards.findOne({"_id": _id});
+    }
+});
+
+Template.cardModal.events({
+    "click #submitCommentButton": function(event){//eslint-disable-line
```

Comment:
To not have an error here, just don't try to name a variable with the same name as a global...
Remove the disable line and just rename to something like eve (that's what I've been doing)